public/validators/main-albatross/NQ08 ACT8 T0FE PTG8 P5RL H2S3 QGXH V15R NVXY.json

```diff
@@ -0,0 +1,14 @@
+  "fee": 0.00,
+  "payoutType": "restake",
+   "contact": {
```

Comment:
Please format the file correctly, there are 3 spaces here and 2 spaces in the line before